src/ckb/actor.rs

```diff
@@ -375,7 +375,7 @@ mod test_utils {
                         request, &tx, &fulfilled_tx
                     );
 
-                    if let Err(e) = reply_port.send(Ok((fulfilled_tx))) {
+                    if let Err(e) = reply_port.send(Ok(fulfilled_tx)) {
```

Comment:
why this was not reported by compiler before this commit,
`#[warn(unused_parens)]` should be enabled by default.

Reply:
I think it's in test package, I found this error when run `cargo test`